cves/CVE-2017-14170.yml

```diff
@@ -64,7 +64,19 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A missing End of File (EOF) check in FFmpeg's Material eXchange Format (mxf)
+  demuxer could cause a Denial of Service attack via a specifically crafted
+  mxf file. Specifically, the mxf file would need to specify an arbitrarily
+  large amount of data in its header, without sufficient data to back it up. The
+  file could also cause this unterminated loop to run multiple times, which could
+  be used to potentially mask the malicious nature of the file by utilizing a
+  smaller number of index entries in the header. The DoS is done by consuming
```

Comment:
I'm not sure I quite understand how decreasing index entries in the header masks the malicious nature of the file. How does the file cause the loop to run multiple times, and how do the index entries influence the execution?

Reply:
The header is what determines the bounds of the loops (at least per my understanding) - if the header says there are 10 entries, the code assumed that there actually were 10 entries, and looped 10 times. As for masking the malicious nature, an arbitrarily large specification in the header would be suspicious and able to potentially be caught out by some basic security analysis. By abusing the specific nature of the loop an attacker could specify a seemingly normal amount of data, and then give a specific amount of data to exploit the code in a way that still causes the DoS attack, but doesn't actually appear suspicious on the surface. I hope that made sense